yw_website/yw_website/apps/website/views.py

```diff
@@ -80,11 +82,45 @@ def my_workflows(request):
     page = request.GET.get("page")
     workflows = paginator.get_page(page)
     host = request.get_host()
-
+    if request.method == "POST":
```

Comment:
I don't think this section is necessary, I don't see a place where we perform a `POST` request on the my_workflows view